tests/counterfactual/test_counterfactual_handler.py

```diff
@@ -693,12 +692,7 @@ def model():
         z = pyro.sample("z", dist.Normal(x + y, 1).to_event(len(event_shape)))
         return dict(w=w, x=x, y=y, z=z)
 
-    if use_biased_preemption:
-        preemption_handler = BiasedPreemptions(
-            actions=preemptions, bias=0.1, prefix="__split_"
-        )
-    else:
-        preemption_handler = Preemptions(actions=preemptions)
+    preemption_handler = Preemptions(actions=preemptions, bias=0.1, prefix="__split_")
 
     with MultiWorldCounterfactual(cf_dim), preemption_handler:
```

Comment:
I'd also consider testing whether the intended preemptions have been implemented and whether their frequency is more or less as intended, maybe along the lines of:

        intervened_y = gather(tr.nodes["y"]["value"], IndexSet(x={1}))
        preempted_y_freq = torch.tensor((intervened_y == 1.0).sum().item() / 1000)
        assert torch.isclose(preempted_y_freq, torch.tensor(0.6), atol=0.09)